src/layouts/social/feed-1/index.tsx

```diff
@@ -39,18 +39,18 @@ export default (): React.ReactElement => {
           style={styles.iconButton}
           appearance='ghost'
           status='basic'
-          icon={ShareIcon}/>
+          accessoryLeft={ShareIcon}/>
```

Comment:
For multiple props components let's use next line for as closing bracket location.
```suggestion
          accessoryLeft={ShareIcon}
        />
```